### DISCUSSION TITLE
Uncommitted changes detected

### DISCUSSION THREAD

**Discussion by R0FGV** [ScoopInstaller/Scoop#5416] [Help and Questions] [answered]
![image](https://user-images.githubusercontent.com/19688305/221869201-7dbb7151-71d3-466d-830f-19e69e5f7bf0.png)
Update scoop encountered this error, because I did not find more solutions to this error

**Comment by rashil2000** [comment, marked as answer]
What's the output of `git status` in the scoop root folder?

**Reply by R0FGV** [reply]
![image](https://user-images.githubusercontent.com/19688305/221873086-48cf4802-e732-4ed1-8ed1-a170f4dd892a.png)

**Reply by R0FGV** [reply]
![image](https://user-images.githubusercontent.com/19688305/221873174-fe90a9f4-4bbd-4f55-8aad-79c568d99c82.png)

**Reply by rashil2000** [reply]
> ![image](https://user-images.githubusercontent.com/19688305/221873086-48cf4802-e732-4ed1-8ed1-a170f4dd892a.png)

Please run `git restore --staged .` in this location and try again.

**Reply by R0FGV** [reply]
Thank you for the perfect solution~~

![image](https://user-images.githubusercontent.com/19688305/221881848-4d528174-b6b3-4b86-bc44-24cd6be7a40c.png)

**Reply by rashil2000** [reply]
You're welcome 😅
